src/proto/grpc/testing/test.proto

```diff
@@ -85,8 +85,21 @@ service LoadBalancerStatsService {
       returns (LoadBalancerStatsResponse) {}
 }
 
+// A service used to obtain real-time stats for verifying LB behavior.
+service LoadBalancerRealTimeStatsService {
+  // Gets the real-time stats for RPCs sent by a test client.
+  rpc GetClientRealTimeStats(LoadBalancerRealTimeStatsRequest)
```

Comment:
Can this just be an RPC method inside of the existing `LoadBalancerStatsService`?

Also, regarding name, it seems like here "real-time stats" = "stats for all RPCs ever sent"? Is that a common meaning of the term "real-time"?

Reply:
Yeah, we can put it into the existing `LoadBalancerStatsService`.

Regarding the naming, doesn't "real-time stats" mean what's being recorded with respect to time as the x-axis? 😄 

Or it could be named as `GetClientAccumulatedStats` if you think that's better.

Reply:
Real time to me indicates there's some sort of real-time response time constraints (like, auto-piloting a jet, https://en.wikipedia.org/wiki/Real-time_computing). OpenCensus's gRPC code has "real-time" metrics that just mean it reports about ongoing streamings RPCs rather than just completed RPCs. Neither of those seem to apply here :) I'm ok if you want to use real time in the name, but maybe include "all RPCs ever sent" in the comment - or just use `GetClientAccumulatedStats`, which does sound clearer to me.

Reply:
Sure, updated to `*AccumulatedStats` and put the RPC definition into `LoadBalancerStatsService`.